html/scripts/get_os.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/bash
+
+cat /etc/issue
```

Comment:
There are some extra characters "\n \l" at the end of OS name, so maybe try to trim them (probably it will be easier to do it in php before `echo`).

Reply:
Done. Also, added some information what they are supposed to be.